staging/src/k8s.io/client-go/tools/portforward/portforward.go

```diff
@@ -404,10 +408,20 @@ func (pf *PortForwarder) handleConnection(conn net.Conn, port ForwardedPort) {
 	err = <-errorChan
 	if err != nil {
 		runtime.HandleError(err)
-		pf.streamConn.Close()
+
+		// If connection was reset by peer, we don't want to close pf.streamConn because that would completely
+		// terminate the port forwarding session. This particular connection is done being handled, but the listener
+		// is still able to accept and handle new connections.
+		if !connectionWasReset.Load() {
+			pf.streamConn.Close()
+		}
 	}
 }
 
+func isUseOfClosedNetworkConnection(err error) bool {
+	return strings.Contains(strings.ToLower(err.Error()), "use of closed network connection")
+}
```

Comment:
this has to be something legacy, you have this error if someone is closing the listenet , with `listener.Close()`

Reply:
I see, there is no graceful shutdown, that is why it fails and silent those errors

Reply:
Yeah, I wasn't completely sure of the reason for that check.  I suspect this could be replaced by `errors.Is(err, net2.ErrClose)` but I'm not 100% certain it is the same thing.